AsyncDisplayKit/Private/_AS-objc-internal.h

```diff
@@ -454,12 +454,13 @@ typedef enum {
         return 1;                                                               \
     }                                                                           \
     -(BOOL)_isDeallocating {                                                    \
-        if (_rc_ivar == -2) {                                                   \
+        __typeof__(_rc_ivar) _prev = __atomic_load_n(&_rc_ivar, __ATOMIC_SEQ_CST); \
+        if (_prev == -2) {                                                      \
             return 1;                                                           \
-        } else if (_rc_ivar < -2) {                                             \
+        } else if (_prev < -2) {                                                \
```

Comment:
I know the code was this way previously, but it seems odd that a local variable would be named with an _ which is the ivar naming convention.